AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/n1ql/pages/n1ql-language-reference/flex-indexes.adoc

To specify that a full-text index should use the keyword analyzer:

> . In the *Search* tab, click either the name of an existing index, or btn:[Create Index] to open the Index Details menu.

Reviewer: ```suggestion
. To open the *Index Details* menu, in the *Search* tab, click either the name of an existing index, or btn:[Create Index].
```

> . Click *General Settings*, open the *Default Analyzer* pull down, and select `keyword`.

Reviewer: ```suggestion
. Click *General Settings*. 
. In the *Default Analyzer* list, select `keyword`.
```

To specify that a full-text index should use the keyword analyzer:

. To open the *Index Details* menu, in the *Search* tab, click either the name of an existing index, or btn:[Create Index].
. Click *General Settings*. 
. In the *Default Analyzer* list, select `keyword`.
+

> image::elixir-flex-fts-index-keyword.png["FTS General Settings menu with Default Analyzer set to keyword",498,552]

Reviewer 2: My preference is for equivalent screenshots to have the same file name across branches, if they're only cosmetically different.
